src/Symfony/Component/Routing/Tests/Generator/Dumper/CompiledUrlGeneratorDumperTest.php

```diff
@@ -290,43 +292,44 @@ public function testTargetAliasWithNamePrefixNotExisting()
 
         file_put_contents($this->testTmpFilepath, $this->generatorDumper->dump());
 
-        $compiledUrlGenerator = new CompiledUrlGenerator(require $this->testTmpFilepath, new RequestContext());
+        $this->expectException(RouteNotFoundException::class);
 
+        $compiledUrlGenerator = new CompiledUrlGenerator(require $this->testTmpFilepath, new RequestContext());
         $compiledUrlGenerator->generate('sub_a');
     }
```

Comment:
@nicolas-grekas after investigating why `testTargetAliasWithNamePrefixNotExisting` are now failing I found out, that both, `CompiledUrlGeneratorDUMPER` and `CompiledUrlGenerator` throw a `RouteNotFoundException`. We are testing `$compiledUrlGenerator->generate('sub_a)` here, but the exception is already thrown in `file_put_contents($this->testTmpFilepath, $this->generatorDumper->dump());` when generating the dump to prepare a file.

So now after moving the `expectedException` method call down, I can say, that we are actually not testing the behavior of `CompiledUrlGenerator` in this test 🤷‍♂️ 

Same applies to `testTargetAliasNotExisting`

Reply:
@nicolas-grekas I can create a separate PR with this failing test, so this PR can be merged. WDYT?

Reply:
sure

Reply:
Done